deploy/helm/values.yaml

```diff
@@ -386,7 +386,7 @@ swoagent:
   enabled: false
   image:
     repository: solarwinds/swo-agent
-    tag: "latest"
+    tag: ""
     pullPolicy: Always
```

Comment:
Do we still need always? `IfNotPresent` should be sufficient now

Reply:
It should. There is no harm in any of those options. I can change that, too, if you want.